### PR TITLE
Adding baked in termux support.

### PR DESCRIPTION
**Pull request by nate-moo**
Currently, the name of the player is set to `android` but we can change that as we see fit.

### CODE REVIEW

File: anime_downloader/util.py [hunk 1]
```diff
@@ -229,10 +229,14 @@ def play_episode(episode, *, player, title):
             f'--referrer={episode.source().referer}', 
             f'--user-agent={get_random_header()["user-agent"]}', 
             episode.source().stream_url])
+    elif player == "android":
+       p = subprocess.Popen(['am', 'start', '-a', 'android.intent.action.VIEW', '-t', 'video/*', '-d', f'{episode.source().stream_url}'])
+       #                      am    start    -a    android.intent.action.VIEW    -t    video/*    -d     "$@"
```
Comment (AbdullahM0hamed) [inline anime_downloader/util.py:234]: Remove this comment

File: anime_downloader/util.py [hunk 2]
```diff
@@ -229,10 +229,14 @@ def play_episode(episode, *, player, title):
...
+    elif player == "android":
+       p = subprocess.Popen(['am', 'start', '-a', 'android.intent.action.VIEW', '-t', 'video/*', '-d', f'{episode.source().stream_url}'])
+       #                      am    start    -a    android.intent.action.VIEW    -t    video/*    -d     "$@" 
     else:
-        p = subprocess.Popen([player, episode.source().stream_url
-                              ])
+       p = subprocess.Popen([player, episode.source().stream_url])
     p.wait()
+    
+
```
Comment (AbdullahM0hamed) [inline anime_downloader/util.py:239]: Too much whitespace

**Review by AbdullahM0hamed — CHANGES_REQUESTED** [review]

**Comment by nate-moo** [comment]
hoh
I made a mistake

**Comment by nate-moo** [comment]
I messed this up, moving to #617

**Pull request closed without merging**